### PR TITLE
Examples added

### PR DESCRIPTION
**Pull request by deevannaveed**
This PR is intended to add the React, Vue and Vanilla JS examples to this repo and add descriptions for each example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+
+## Examples in this repo
+### [ReactJS Example](https://github.com/Akkadu/multilingual-sdk-examples/react-example)
+### [VueJS Example](https://github.com/Akkadu/multilingual-sdk-examples/vue-example)
+### [VanillaJS Example](https://github.com/Akkadu/multilingual-sdk-examples/vanillajs-example)
+
+## Demo
+### [Live Demo](https://rsi-akkadu-react-demo.netlify.app/interpretation-player)
+
+## Live SDK key tester
```
Comment (abelbordonado) [inline README.md]: Test your Key.
The link is broken here

File: react-example/src/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (abelbordonado) [inline react-example/src/index.js:16]: What is this one all about?

File: react-example/src/reportWebVitals.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry);
```
Comment (abelbordonado) [inline react-example/src/reportWebVitals.js]: `reportWebVitals`  seems interesting but is only used in the React example.
I'm not aware of this naming `getCLS`. Is a little technical here.

File: README.md (hunk 1 above)
Comment (deevannaveed) [reply README.md]: Updated it.

File: react-example/src/reportWebVitals.js (hunk 3 above)
Comment (deevannaveed) [reply react-example/src/reportWebVitals.js]: I have removed WebVitals, no need to use in this project.

**Review by abelbordonado — APPROVED** [review]
Approved! :D Great job, is pitty are gonna be head less soon lol 🔪

**Pull request merged**